### PR TITLE
point to 3rd party tools for compose

### PR DESCRIPTION
**Pull request by muayyad-alsadi**
Signed-off-by: alsadi <alsadi@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,9 @@ This project tests all builds against each supported version of Fedora, the late
   Podman can also generate Kubernetes YAML based on a container or Pod (see
   [podman-generate-kube](https://github.com/containers/libpod/blob/master/docs/podman-generate-kube.1.md)),
   which allows for an easy transition from a local development environment
-  to a production Kubernetes cluster.
+  to a production Kubernetes cluster. Third-party tools might support `docker-compose` format
+  like [kompose](https://github.com/kubernetes/kompose/)
+  and [podman-compose](https://github.com/muayyad-alsadi/podman-compose).
```
Comment (TomSweeneyRedHat) [inline README.md]: Suggest: "If Kubernetes does not fit your requirements, there are other third-party tools that support the `docker-compose` format such as [kompose](https://github.com/kubernetes/kompose/)
  and [podman-compose](https://github.com/muayyad-alsadi/podman-compose) that might be appropriate for your environment."

**Comment by TomSweeneyRedHat** [comment]
I tried to reword it a bit to take away the impression that Podman has been thoroughly tested with these tools.  I think the information is probably good to have for the end user as another option for their arsenal.

**Comment by muayyad-alsadi** [comment]
@TomSweeneyRedHat suggestion applied

**Comment by rhatdan** [comment]
/ok-to-test 
/approve
@muayyad-alsadi Could you squash your PRs.

git rebase -i origin 
Use Fixup on the second patch
git push --force

**Comment by muayyad-alsadi** [comment]
it seems that I squashed Jhon Honce large work into my work?

**Comment by TomSweeneyRedHat** [comment]
LGTM, thanks @muayyad-alsadi !
The tests are failing and I can't imagine are due to these changes.  @mheon do you know what's up with them?

**Comment by mheon** [comment]
Restarted failing tests, let's see if they're flakes
LGTM once tests go green

**Comment by rhatdan** [comment]
@muayyad-alsadi You need to rebase this PR and it will get merged.  The rebase will fix the CI errors.

git pull origin master
git rebase -i origin
git push --force

**Comment by muayyad-alsadi** [comment]
@rhatdan  I guess it needs LGTM

**Comment by mheon** [comment]
Merging
/lgtm

**Pull request merged**